src/OVAL/results/oval_resultTest.c

```diff
@@ -465,6 +465,132 @@ static inline oval_result_t _evaluate_sysent(struct oval_syschar_model *syschar_
 	}
 }
 
+struct oval_item_statuses {
```

Comment:
Perhaps move all connected with this to a separate C file.

Regarding the name of this structure, I would like to ask someone who speaks English better than me, because "stats" would mean for me "statistics", which is not this case.